src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx

```diff
@@ -20,7 +21,12 @@ export const AccountExplorers = memo(({ accountId, chain, children }: Props) =>
   return (
     <Popover align="end" dialog testId="AccountExplorers">
       <Popover.Trigger>
-        <IconButton name="details" className="text-icon-default" onClick={(e) => e.stopPropagation()} />
+        <IconButton
+          name="details"
+          className="text-icon-default"
+          testId={TEST_IDS.MAIN.INFO_BUTTON}
```

Comment:
I think ui-entities and ui-kit should be free from app-centric constants like this. It's better to get testId from component props and have reasonable default, usually it's component name

Reply:
Is that what you meant?
https://github.com/novasamatech/nova-spektr/pull/2668/commits/d6a12e55d588fc3eb8b699293ed50b70a3df73e8